test/runtests.jl

```diff
@@ -83,6 +83,16 @@ end
     @test C(-Inf..1, IntervalBox(0.5..1.5, 2)) == IntervalBox(0.5..0.5, 2)
 end
 
+@testset "Contractors for functions having parameters" begin
+
+    @parameters r
+    vars =@variables x
+    f(x) = r*x*(1-x)
+
+    C = Contractor(vars, f)
+    @test C(-Inf..1, IntervalBox(0.5..1.5), 2) == IntervalBox(0.5..1.5) #parameter r is set 2
```

Comment:
I'm not sure if this is the best order for the arguments to `C`.
You should include a test case with a different value of the parameter to show that the result is different!

Reply:
And the result should be different from the input box.

Reply:
The order I decided is C( ConstraintInterval, IntervalBox of variables, parameters ). U can suggest any other if you want.